messaging/copr_messaging/private/hierarchy.py

```diff
@@ -184,6 +184,8 @@ def usernames(self):
 
 
 class _BuildChrootMessage(_BuildMessage):
+    severity = message.DEBUG
```

Comment:
??? Why?

Reply:
The chroot messages are always paired with a build message, which means two notifications for basically the same thing in FMN. By setting the chroot severity to DEBUG (which seems the least notify-worthy of the two), it will not generate a notification in FMN by default.

By setting the severity in a message, you can hint people who subscribe to your events which events are important, which are not, and which require more attention, pretty much like log entries.

Does this answer your question?